example_scripts/ANChor_plots_of_NSea_wvel.py

```diff
@@ -25,17 +25,22 @@
 dir_nam = '/projectsa/NEMO/jelt/AMM60_ARCHER_DUMP/AMM60smago/EXP_NSea/OUTPUT/'
 fil_nam = 'AMM60_1h_20120204_20120208_NorthSea.nc'
 
-chunks = {"x":10, "y":10, "time_counter":10}
-sci_w = coast.NEMO(dir_nam + fil_nam, 
-                 dom_nam, grid_ref='w-grid', multiple=True, chunks=chunks)
+config = "/work/jelt/GitHub/COAsT/example_files/example_w_nemo_config.json"
```

Comment:
Before this gets merged, can we change this to something generic (i.e. using "~") and platform-independent (i.e. using Path)?

Reply:
@jpolton @davbyr how would you like the example scripts to access the required content? I could make it all relative like Matt suggests. More like the following;

``` python
dir_nam = Path("/projectsa/accord/GCOMS1k/")
dom_location = dir_name / "INPUTS" / "BLZE12_C1"
fil_location = dir_name / "OUTPUTS" / "BLZE12_02" / "2015"

dom_nam = "domain_cfg.nc"
fil_nam = "BLZE12_1h_20151101_20151130_grid_T.nc"

config_t = Path("~/example_files/example_t_nemo_config.json")
```

Reply:
Good question. This is a folder of tested scripts that forms the basis of a teach-yourself COAsT with some worked examples. Files ending in `_tutorial.py` are more polished. Other scripts were knocked together to get someone started with a real world use case. The above `config` example was me getting the code to work with the new `json` file structure (that I don't yet fully understand).

How should the path/file names appear? Since these are not exposed in the `conda install ` it is more OK for the path names to be specific and not working for everyone in the world. Especially if  "For example" comment was added.

Ideally these paths would be updated to access data only available from the ERDAP server, but we were a bit lazy with prepping data for it. I think that that is the appropriate direction of travel though.

This action is probably worth a ticket

Reply:
https://jira.ceh.ac.uk/secure/RapidBoard.jspa?rapidView=447&view=detail&selectedIssue=BODCDMSYS-3710